packages/editor-ui/src/views/NodeView.vue

```diff
@@ -172,6 +173,7 @@ export default mixins(
 	titleChange,
 	workflowHelpers,
 	workflowRun,
+	externalHooks,
```

Comment:
Is not in alphabetical order.

Reply:
@janober could probably setup husky pre-commit hooks to handle sorting imports and possibly failing commits until things that cannot be auto fixed (like using camelCase) are fixed manually by the committer

Reply:
For the backend modules, the rules are already in place and everything got cleaned up and is fine. On the frontend, however, we did not look into that yet. So agree we probably should do that soon.